common/static/common/templates/discussion/new-post.underscore

```diff
@@ -9,20 +9,20 @@
     <% } %>
     <ul class="post-errors" style="display: none"></ul>
     <div class="forum-new-post-form-wrapper"></div>
-    <% if (cohort_options) { %>
+    <% if (group_options) { %>
```

Comment:
I fixed the bug-- it was an edge case when more than one topic had "General" in it.